src/solvers/refinement/string_constraint_generator_indexof.cpp

```diff
@@ -131,10 +219,16 @@ exprt string_constraint_generatort::add_axioms_for_last_index_of_string(
   symbol_exprt contains=fresh_boolean("contains_substring");
 
   // We add axioms:
-  // a1 : contains => |substring| >= length &&offset <= from_index
-  // a2 : !contains => offset=-1
-  // a3 : forall 0 <= witness<substring.length,
-  //        contains => str[witness+offset]=substring[witness]
+  // a1 : contains => |substring| >= length && offset <= from_index
```

Comment:
Doesn't appear to match the code? Looks like below it says `|substring| + offset > |str|`?